dev/release/src/release.ts

```diff
@@ -352,6 +352,16 @@ These steps must be completed before this PR can be merged, unless otherwise sta
 
 ${actionItems.map(item => `- [ ] ${item}`).join('\n')}
 
+${
+    notPatchRelease
+        ? ''
+        : `### :information_source: How to update CHANGELOG
+- [ ] I understood the convention of our changelog format. [learn more](https://keepachangelog.com/en/1.0.0/)
```

Comment:
This doesn't look like our changelog format 🤔 Can we just assume to follow the existing changelog format?

Reply:
I was assuming our format is derived from the keepachangelog format. They're close enough besides we don't have the date in the H2. Also, change types are slightly different, but every project has its own flavour.

But yeah, I think we can just ask the Release Captain to review our current changelog to learn about our convention.

```suggestion
- [ ] I understood the convention of our changelog format. [learn more](https://github.com/sourcegraph/sourcegraph/blob/main/CHANGELOG.md)
```